src/Request.js

```diff
@@ -85,6 +85,10 @@ define([
 		//		If this is not specified, the range will
 		//		included with a RQL style limit() parameter
 
+		// requestOptions: Object
+		//		Additional options to pass to dojo/request
+		requestOptions: {},
```

Comment:
I would probably suggest initializing this to `null` just to avoid potential footguns...

e.g.:

```js
var store1 = new Request(...);
store1.requestOptions.foo = 'bar';

// requestOptions.foo also set here because the object is shared at the prototype level
var store2 = new Request(...); 
```